### DISCUSSION TITLE
Add support for resourceTags in "copilot env deploy" and other commands where feasible

### DISCUSSION THREAD

**Discussion by tsingh2k15** [aws/copilot-cli#3872] [Ideas]
Is it feasible to add support for `resourceTags` for `copilot env deploy` command at https://github.com/aws/copilot-cli/blob/mainline/internal/pkg/cli/env_deploy.go#L29? This thought is similar to what is currently available for `copilot svc deploy` command/s at https://github.com/aws/copilot-cli/blob/mainline/internal/pkg/cli/svc_deploy.go#L40.

In general, it could be beneficial to have `resourceTags` option on multiple cli commands(where feasible) that may allow users to apply custom tags on all the resources which copilot creates or adding tags on resources already created by copilot. Having custom tags is generally a hard requirement at many organizations where users may be expected to tag all the resources. Thoughts?

**Comment by efekarakus** [comment]
That sounds great @tsingh2k15 ! Makes sense to me, I'll convert the discussion into a feature request.

Edit: Moved it to https://github.com/aws/copilot-cli/issues/3874
